pkg/specgenutil/volumes_test.go

```diff
@@ -15,39 +15,39 @@ func Test_validChownFlag(t *testing.T) {
 		{
 			name: "U true",
```

Comment:
...but this description should now be `lower-case true` or something similar...

Reply:
renamed to `lower-case true`